### ISSUE TITLE
networkx.circular_layout returns different coordinates between Ubuntu and OS X

### ISSUE THREAD

**Issue by canavandl** [networkx/networkx#2135]
Using the below code example, Ubuntu (16.04) and OS-X (10.11 El Capitan) return different coordinates for graphs passed to the `circular_layout` method. OS-X coordinates have a range of 0 to 1 (for x and y space) while Ubuntu returns a range from -1 to 1 (for x and y space)

```python
import networkx as nx
graph = nx.Graph()
graph.add_nodes_from(["one", "two", "three"])
```

Result on Ubuntu
```bash
$ nx.circular_layout(graph)
>  {'one': array([-0.5      , -0.8660254]),
    'three': array([ 1.,  0.]),
    'two': array([-0.5      ,  0.8660254])}

$ conda list | grep -e networkx -e pandas -e numpy
>  networkx                  1.11                     py27_0    defaults
   numpy                     1.10.4                   py27_0    defaults
   pandas                    0.17.1              np110py27_0    defaults
```
 Result on OS-X
```bash
$ nx.circular_layout(graph)
>  {'one': array([  8.60318892e-08,   0.00000000e+00], dtype=float32),
    'three': array([ 0.86602539,  0.5       ], dtype=float32),
    'two': array([ 0.,  1.], dtype=float32)}

$ conda list | grep -e networkx -e pandas -e numpy
>  networkx                  1.11                     py27_0    defaults
   numpy                     1.10.4                   py27_1    defaults
   pandas                    0.17.1              np110py27_0    defaults
```

**Comment by hagberg** [comment]
That is odd.  There were some changes in the scalings at some point.  Would you double check the versions by running
$ python -c "import networkx; print(networkx.__version__)"

**Comment by dschult** [comment]
Just to verify.... can you print  nx.__version__?
We did change the circular layout from [0,1] to [-1,1] and while they might
both be 1.11 they might have different version numbers.  That would be
confusing so hopefully not the case, but I just want to make sure.
Dan

**Comment by canavandl** [comment]
oh goodness...

It looks like the conda package I was using for networkx=1.11 on OSX is actually a mistagged build of networkx=1.7.

![screen shot 2016-05-18 at 4 53 51 pm](https://cloud.githubusercontent.com/assets/2198981/15376277/93a00d64-1d19-11e6-9345-905972ec2454.png)

I don't know where the mistagged package is from - I deleted it and reinstalled from conda main and received the correct version. Maybe it's a since resolved problem with conda? (conda caches packages to share between environments and I don't know when I downloaded the original).

Anyway - I'll mark as closed. Thanks!
